build/scripts/single-source-security-fixes.sh

```diff
@@ -7,15 +7,28 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+# Single-source the release notes Fixed security issues section from Red Hat Security Data API.
```

Comment:
could rename this to modules/release-notes/snip-common-vulnerabilities-and-exposures.sh so it's located in the same place as the other files... might make it more obvious you have to run the script to regenerate adoc.